vrk-kpa/suomidigi
# SUOPA-655 Suomidigi Event

To test: 
- `make fresh`
- Go to https://suomidigi.docker.sh/node/add/event and fill in all the details
- After saving the node, check that the layout works for mobile and desktop versions.
